source/1_engine/rex_engine/include/rex_engine/diagnostics/logging/internal/details/registry.h

```diff
@@ -30,7 +30,7 @@ namespace rexlog
     class Registry
     {
     public:
-      using LogLevels = rex::DebugHashTable<rex::DebugString, level::LevelEnum>;
+      using LogLevels = rsl::unordered_map<rsl::string_view, level::LevelEnum>;
```

Comment:
Aren't these stored, so making it a string_view seems kinda dangerous to me

Reply:
they come directly from the commandline, if none are passed in it uses the global level instead

Reply:
Btw isn't this the same type

Reply:
what's the difference?

Reply:
if no log levels are passed on the command line, the global one is used

Reply:
I mean isn't this the same type as the using I mentioned before

Reply:
no, well this is the using you're talking about I think